src/Jadu/Pulsar/Twig/Extension/HelperOptionsModifierExtension.php

```diff
@@ -40,7 +40,7 @@ public function modifyOptions($options)
         }
 
         // Check if errors are present in options
-        if (array_key_exists('error', $options) and count($options['error']) > 0) {
+        if (array_key_exists('error', $options) and is_array($options['error']) and count($options['error']) > 0) {
```

Comment:
I'd swap these if only for readability sake, make it read as the `is_array` is the first thing we check

Reply:
Done